lms/urls.py

```diff
@@ -820,10 +820,16 @@
 
 if settings.FEATURES.get('ENABLE_OAUTH2_PROVIDER'):
     urlpatterns += (
+        # These URLs dispatch to django-oauth-toolkit or django-oauth2-provider as appropriate.
+        url(r'^oauth2/', include('lms.djangoapps.oauth_dispatch.urls')),
+        # These URLs contain the django-oauth2-provider default behavior (for views that have not
+        # been converted to dispatching views
         url(r'^oauth2/', include('edx_oauth2_provider.urls', namespace='oauth2')),
+        # These are the standard django-oauth-toolkit views.  They should not be used publicly,
+        # but exist for code that requires the oauth2_provider namespace.
+        url(r'^_o/', include('oauth2_provider.urls', namespace='oauth2_provider')),
```

Comment:
Which existing internal code still needs the `oauth2_provider` namespace?  Shouldn't we update those callers?  Is there a follow-up ticket to do so?